src/content/0.0/preface.tex

```diff
@@ -40,7 +40,7 @@
 definition of the category itself. And I will argue strongly that
 composition is the essence of programming. We've been composing things
 forever, long before some great engineer came up with the idea of a
-subroutine. Some time ago the principles of structural programming
+subroutine. Some time ago the principles of structured programming
```

Comment:
Why this change?

Reply:
I think the term "structured programming" is more widely used, as @bigaure pointed out in #265.